### DISCUSSION TITLE
User is not signed into GitHub and/or doesn't have Copilot chat access.

### DISCUSSION THREAD

**Discussion by Phylum123** [community/community#56226] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I was granted access to the github copilot x beta. After installing it from the download link in the email I got, when I open visual studio I get a bar at the top that says that I need to sign up for the private beta. When I click "sign up for the private beta", it goes to a github webpage saying "You're in! The GitHub Copilot chat private beta is already enabled on your account!". I've tried erasing my github account from vs 2022 and re-adding it, and restarting vs, several times. Regardless, the output window shows this, every time:

[Conversations Information] Activated Conversations version 0.1.651-beta+8606258f14.
[Conversations Information] [ServiceTarget] Conversations service is running. PID: 73388
[Conversations Information] [CopilotAuthTokenManager] Need to get an auth token.
[Conversations Information] [CopilotAuthTokenManager] Status Code: 403, Reason Phrase: Forbidden
[Conversations Information] [CopilotAuthTokenManager] Response Content: HasToken: False, ChatEnabled: , ExpiresAt: 0, RefreshIn: 0, SKU: 
[Conversations Error] [CopilotAuthTokenManager] Response didn't contain a token.
[Conversations Error] [Conversations] Exception in VerifyUserHasChatAccessAsync: System.UnauthorizedAccessException: User is not signed into GitHub and/or doesn't have Copilot chat access.

**Comment by breadboi** [comment]
I had the same issue and was able to get mine working after signing into my GitHub account via the bubble in the top right of your Visual Studio.
![image](https://github.com/community/community/assets/32111055/7ea436bc-859c-4ccb-b8a5-f16d2ea10191)

**Reply by Phylum123** [reply]
I already was signed in to my github account. I deleted and re-added it then closed and opened visual studio, same problem. Thank you for the suggestion though :).

**Comment by Phylum123** [comment]
I figured it out. My trial for Copilot had ended. Had to buy a subscription, then it worked.
